src/Camera.cpp

```diff
@@ -8,7 +8,10 @@
 #include "OptimizationCeres.h"
 #include "logger.h"
 
-Camera::Camera() {}
+Camera::Camera(const int cam_idx, const int distortion_model) {
+  cam_idx_ = cam_idx;
+  distortion_model_ = distortion_model;
+}
```

Comment:
Camera::Camera(const int cam_idx, const int distortion_model) : cam_idx_(cam_idx), distortion_model_(distortion_model) {}; 
 
can be an option

Reply:
Incorporated this feedback in the latest commit